### PR TITLE
Update to kernel-4.19.1 and rework patches

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

**Comment by marmarek** [comment]
What's left for making spec.in? If that's not too much work, maybe it worth doing it here too?

**Comment by fepitre** [comment]
Yes it worth to do it. Up to what I remembered when doing all the spec.in, it was that there is several things to solve related to actions done in the current spec. I will investigate and do it.

File: kernel.spec.in [hunk 1]
```diff
@@ -41,7 +35,11 @@
 %define setup_config --enable CONFIG_DEBUG_INFO --disable CONFIG_DEBUG_INFO_REDUCED
 %endif
 
-Name:           kernel%{name_suffix}
+%if "%{?name_suffix}" != ""
+Name:           kernel-%{name_suffix}
```
Comment (marmarek) [inline kernel.spec.in]: Why not keep name_suffix "-latest" (including the dash) and avoid this %if?

File: kernel.spec.in [hunk 2]
```diff
@@ -520,5 +526,4 @@ exit 0
 %attr(0644, root, root) %vm_install_dir/vmlinuz
 %vm_install_dir/modules
 
-
-%changelog
+@CHANGELOG@
```
Comment (marmarek) [inline kernel.spec.in]: Shouldn't this be:
```
%changelog
@CHANGELOG@
```
?

**Comment by fepitre** [comment]
Oh yes for both. I will correct.

**Pull request merged**